### DISCUSSION TITLE
RELEASE Window+.bsp

### DISCUSSION THREAD

**Discussion by Galvinh0slk4** [nzp-team/nzportable#1066] [Map Releases]
[DOWNLOAD](https://download1326.mediafire.com/v1tw41u0no0gSdHQNs5-28SozzaaPlvPdDgX0e5wDs79j0vf65VFalnjLQ2hEkRJbiqufdRSC4M7I7jPGE1Cpy2TzQsV16cX3hyNMbcUAhZw0op8V3jatzJ096HTpRgRocelaqv71U0eOlJL0oZeL5FgOkvwcNGBAiUoQHKtr5vnIyw5/k3hctomvgcoiovb/Window%2B.zip)

this is supposed to be a remake of the map I had made earlier, lose the file because I had to format the PC, the map is bug-free, everything revised, if you find it please inform, and I will answer you.

PRINTS
![Windows+](https://github.com/user-attachments/assets/8408aaf7-0bd6-492f-b28e-59f8bc7f1d21)
![fte-20241228153050-0](https://github.com/user-attachments/assets/51f474cc-6237-4bfd-92e7-3abf5c0c6a21)

**Comment by irandiaz** [comment]
Hey, where should the power be?

**Reply by Useless3** [reply]
It's right next to the Mystery Box. To unlock it, you have to find some teddy bears. I've only seen two so far; I'm not sure if I did it in the right order or if there are more teddy bears. One is on top of the tree behind the double tap, and the other is behind the car on the right of the street lamp. I'm yet to find a third one.

**Reply by Useless3** [reply]
Update: It's definitely not the order, and I don't actually know if the teddies actually unlock power, but considering that power is locked, and there's nothing else to unlock, it's most likely power that the teddy bears unlock.

**Comment by irandiaz** [comment]
![Uploading Screenshot_20250314_214915_org_ppsspp_ppssppgold_PpssppActivity.jpg…]()
